test/browser/home-page.test.jsx

```diff
@@ -0,0 +1,30 @@
+var should = require('should');
+var React = require('react/addons');
+var TestUtils = React.addons.TestUtils;
+
+var stubContext = require('./stub-context.jsx');
+var stubBlogFeedLoader = require('./stub-blog-feed-loader.js');
+var HomePage = require('../../pages/home.jsx');
+
+describe("HomePage", function() {
+  var homePage, blogSection;
+
+  beforeEach(function() {
+    homePage = stubContext.render(HomePage);
+    blogSection = stubContext.render(HomePage.BlogSection);
+  });
+
+  describe("blogSection", function() {
+    it("should display featured post", function() {
+      blogSection.getDOMNode().querySelector(".featured-post .entry-title").textContent.should.not.eql("");
+    });
+    it("should display 3 other latest posts", function() {
+      blogSection.getDOMNode().querySelectorAll(".recent-posts .post-title").length.should.equal(3);
+    });
```

Comment:
Hey @toolness!

( I created this PR against my own PR so that I(or we) can easily compare the code changes. )
I'm having some tough time writing tests for this page.  The above 2 test assertions are failing and I can't come up with a solution. Hellllp :confounded: Is it because this test get run *before* the actual feed data arrives?